app/javascript/react/screens/App/Mappings/components/DeleteInfrastructureMappingConfirmationModal/DeleteInfrastructureMappingConfirmationModal.js

```diff
@@ -75,7 +66,10 @@ const DeleteInfrastructureMappingConfirmationModal = ({
           {mappingToDelete &&
             unUsedMappingInPlans(mappingToDelete, notStartedTransformationPlans, finishedWithErrorTransformationPlans)}
         </h4>
-        <h4>{mappingToDelete && displayDeleteMessage(mappingToDelete)}</h4>
+        <h4>
+          {mappingToDelete &&
+            sprintf('Are you sure you want to delete the infrastructure mapping %s ?', mappingToDelete.name)}
```

Comment:
We still need the `__(...)` gettext call for i18n here. and we also lose the `<strong>` around the name. maybe instead we should just do:
```
<h4>
  {__('Are you sure you want to delete the infrastructure mapping ')}
  <strong>{mappingToDelete && mappingToDelete.name}</strong>
  {' ?'}
</h4>
```

Reply:
The thing is that we need to avoid doing string math in gettext strings as much as possible.

In this particular example, it would not be obvious that the string is an actual question, since the question mark comes separately. That would make it impossible to translate it correctly into Spanish language for example (¿ at the beginning of interrogative sentence).

What do you think about the following?

![infra-mapping-delete-01](https://user-images.githubusercontent.com/6648365/54025293-606c7380-419a-11e9-8c25-76cf47651083.png)

Reply:
That looks good to me. @vconzola are you ok with this slight design change? (previously the name of the mapping was a part of the sentence).